src/EnergyPlus/ConvectionCoefficients.hh

```diff
@@ -863,6 +863,14 @@ namespace ConvectionCoefficients {
                          Real64 RoofArea,
                          Real64 RoofPerimeter);
 
+    void CalcASTMC1340ConvCoeff(EnergyPlusData &state,
+                                int const SurfNum,                  // surface number for which coefficients are being calculated
+                                Real64 const SurfaceTemperature,    // Temperature of surface for evaluation of HcIn
+                                Real64 const ZoneMeanAirTemperature // Mean Air Temperature of Zone
+    );
+
+    Real64 CalcASTMC1340ConvCoeff(int const SurfNum, Real64 const Tsurf, Real64 const Tair, Real64 const Vair, Real64 const Tilt);
```

Comment:
Hmmm, are the other convection coefficient functions named in this same pattern where the function names are overloaded?  I mean it's clear which one you are trying to use by what arguments you pass, of course, but it does kinda look a little funny.  I don't think we need to change anything necessarily right now, especially if that is the pattern being used across other convection functions.  But it is a little funny.

Reply:
@Myoldmopar I remember when the convection algorithm is the same for an entire zone, there's a function to cycle through the surfaces and evaluate the convection coefficients by calling the base function. And if the convection algorithm is adaptive or schedule, then there are calls directly to the base function. I did not remember that some of these use the same name for both - kind of confusing. e.g. `CalcCeilingDiffuserIntConvCoeff` and `CalcASHRAESimpleIntConvCoeff`.

Reply:
I tried to follow `CalcASHRAESimpleIntConvCoeff` functions, and it seems like these are the only other functions named in the same pattern. (https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/ConvectionCoefficients.cc#L2360-L2446) I could change it to be called directly to the base function if needed.